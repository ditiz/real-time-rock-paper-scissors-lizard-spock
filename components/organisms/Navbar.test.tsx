import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import NavbarOrganism from "./Navbar"

it("renders NavbarOrganism unchanged", () => {
  const { container } = render(<NavbarOrganism />)
  expect(container).toMatchSnapshot()
})

it("should have logo", () => {
  // TODO: create logo and check it here
})

it("should have links", () => {
  render(<NavbarOrganism />)
  expect(screen.getAllByRole("link")).toHaveLength(2)
})
