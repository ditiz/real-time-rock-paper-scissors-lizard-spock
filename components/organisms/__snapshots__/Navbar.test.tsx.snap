// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavbarOrganism unchanged 1`] = `
<div>
  <nav
    class="navbar"
  >
    <div
      class="navbar__logo"
    />
    <div
      class="navbar__links"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/Rules"
      >
        Rules
      </a>
    </div>
  </nav>
</div>
`;
